src/usercmd.c

```diff
@@ -102,6 +102,7 @@ static struct
     {EXPAND_BREAKPOINT, "breakpoint"},
     {EXPAND_SCRIPTNAMES, "scriptnames"},
 #endif
+    {EXPAND_DIRS_IN_CDPATH, "dir_in_path"},
```

Comment:
This is not documented in `:h getcompletion()` and `:h command-complete`

Reply:
Thanks, I had completely missed that.